frontend/src/pages/login/Login.scss

```diff
@@ -55,9 +67,12 @@
         }
       }
 
-      &__submit {
```

Comment:
Not sure what this does, but I'm assuming that it's ok to modify it...

Reply:
Nothing happened here, its just so we have the submit button to the right and we can have something else on the right.